src/index.html

```diff
@@ -7,8 +7,42 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap&subset=cyrillic,cyrillic-ext" rel="stylesheet">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <div class="logo">
+      <a class="logo-link" href="">
+        <img src="images/logo.png" alt="moyo">
+      </a>
+    </div>
+    <nav class="navigation">
+      <ul class="links">
+        <li class="link">
+          <a class="link-sign is-active" href="">Apple</a>
```

Comment:
```suggestion
          <a class="link-sign is-active" href="/">Apple</a>
```
аттрибут не может быть пустым

Reply:
ок, добавлю везде